main.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 23 22:33:29 2019
+
+@author: j-bd
+"""
+import os
+import logging
+
+import pandas as pd
+
+import pneumonia_functions
+import pneumonia_detection
+import constants
+
+
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)
+
+def pre_trainning(args):
```

Comment:
```suggestion
def pre_training(args):
```

Reply:
You're not `pre-traning`, you're `prepararing the training`, it does not mean the same thing. `pre-traning` mean performing a first training